templates/ai/mlops-workshop.html

```diff
@@ -0,0 +1,329 @@
+              </p>
+              <ul class="p-list--divided">
+                <li class="p-list__item has-bullet">
+                  Get guided, hands-on experience with the latest open-source ML tooling, adjusted to your use case and needs
+                </li>
+                <li class="p-list__item has-bullet">
+                  Live Q&A session with AI and MLOps experts to help you accelerate time-to-market and unblock your projects
+                </li>
+                <li class="p-list__item has-bullet">Access a library of written content to further upskill after the workshop</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <hr class="p-rule--muted" />
+        <div class="p-section--shallow">
+          <div class="row">
+            <div class="col-3">
+              <h3 class="p-heading--5">
+                Make the best use
+                <br class="u-hide--small u-hide--medium" />
+                of existing infrastructure
+              </h3>
+            </div>
+            <div class="col-6">
+              <div class="p-section--shallow">
+                <div class="p-image-container--16-9 is-highlighted">
+                  {{ image(url="https://assets.ubuntu.com/v1/eb833a2e-existing-infrastructure.png",
+                                    alt="",
+                                    width="1800",
+                                    height="1013",
+                                    hi_def=True,
+                                    loading="lazy") | safe
+                  }}
+                </div>
+              </div>
+              <p>
+                Optimize your existing infrastructure to maximize efficiency and performance for your AI/ML workloads. Learn about:
+              </p>
+              <ul class="p-list--divided">
+                <li class="p-list__item has-bullet">Data collection improvement opportunities</li>
+                <li class="p-list__item has-bullet">GPU optimization techniques and strategies</li>
+                <li class="p-list__item has-bullet">Model packaging and distribution best practices</li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>MLOps workshop structure</h2>
+        </div>
+        <div class="col">
+          Pick one of our 4 workshops tailored to different use cases, and Canonical will help you design your architecture and long-term AI strategy.
+        </div>
+      </div>
+    </div>
+    <div class="p-section--shallow">
+      <div class="p-equal-height-row">
+        <div class="p-equal-height-row__col">
+          <div class="p-equal-height-row__item">
+            <div class="p-image-container is-highlighted">
```

Comment:
This won't have an effect yet cause prod still isn't using latest vanilla (putting a pr up for this now), but once it is the aspect ratio for these will be responsive
```suggestion
            <div class="p-image-container--3-2-on-small p-image-container--square-on-medium p-image-container--2-3-on-large is-highlighted">
```